src/main/java/core/basesyntax/dao/impl/ShopParserImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.ShopParser;
+
+public class ShopParserImpl implements ShopParser {
+    private static final String END_LINE = System.lineSeparator();
+
+    @Override
+    public String[] parse(String dateFromFile) {
```

Comment:
```suggestion
    public List<Fruitransaction> parse(List<String> dateFromFile) {
```
lets this service parse Strings into fruitTransactions

Reply:
not fixed